### DISCUSSION TITLE
HistoryGraph Meldung?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1296] [Q&A]
Wann kommt die Meldung, wenn kein Wert mehr in der influxdb ist?
![image](https://user-images.githubusercontent.com/84818934/145907058-1499f083-e095-45e2-8db9-2aa2adb7b07f.png)

**Comment by Zefau** [comment]
Kommt die Meldung noch? Die sollte nur erscheinen wenn der Adapter nicht erreichbar ist. Wenn Werte geladen werden und die Meldung kommt ist das falsch. Ich meine aber, dass ich das behoben hab.

**Comment by mcuiobroker** [comment]
Nein es kommt keine Meldung mehr.
